ui/src/dashboards/components/TableOptions.tsx

```diff
@@ -101,48 +92,37 @@ export class TableOptions extends PureComponent<Props, {}> {
     handleUpdateTableOptions({...tableOptions, timeFormat})
   }
 
-  handleToggleTimeAxis = () => {}
+  onToggleVerticalTimeAxis = verticalTimeAxis => () => {
+    const {tableOptions, handleUpdateTableOptions} = this.props
+    handleUpdateTableOptions({...tableOptions, verticalTimeAxis})
+  }
 
   handleToggleTextWrapping = () => {}
 
   handleColumnRename = column => {
     const {handleUpdateTableOptions, tableOptions} = this.props
     const {columnNames} = tableOptions
-    const updatedColumns = columnNames.map(
-      op => (op.internalName === column.internalName ? column : op)
-    )
+    const updatedColumns = columnNames.map(op => (op.internalName === column.internalName ? column : op))
     handleUpdateTableOptions({...tableOptions, columnNames: updatedColumns})
   }
 
   render() {
-    const {
-      tableOptions: {timeFormat, columnNames: columns},
-      onResetFocus,
-      tableOptions,
-    } = this.props
-
-    const TimeAxis = 'vertical'
+    const {tableOptions: {timeFormat, columnNames: columns, verticalTimeAxis}, onResetFocus, tableOptions} = this.props
```

Comment:
this is obviously more than a prettier change but it looks like it should be on separate lines.